src/eth_provider/database/ethereum.rs

```diff
@@ -356,4 +356,164 @@ mod tests {
             Some(mock_pending_transaction.tx)
         );
     }
+
+    fn u64_to_b256(value: u64) -> B256 {
+        let mut bytes = [0u8; 32];
+        bytes[24..32].copy_from_slice(&value.to_be_bytes());
+        B256::from(bytes)
+    }
```

Comment:
Here you can use different strategies to transform a u64 to a B256 but please let us not do a util for that:
- `B256::from_str("0xbeefcafebeefcafebeefcafebeefcafebeefcafebeefcafebeefcafebeefcafa")`
- `B256::from(U256::from(2423))`
- ....